docs/release_notes.rst

```diff
@@ -389,3 +389,8 @@ Commit log
 The above lists are the highlights of what changed. For a full list of the changes since 2.1.8,
 please see the
 `changelog <https://github.com/fedora-infra/bodhi/compare/2.1.8...2.2.0>`_.
+
+Help
+^^^^
+
+Bodhi's documentation is available online: https://bodhi.fedoraproject.org/docs
```

Comment:
We don't want this text in the release notes. What I meant was for you to add a release note about this change at the top of the file in the develop/Features section. Just a quick sentence about how the footer has a documentation link now or something, and a link to the issue number this PR is addressing.